packages/application/src/application/services/pull-request.types.ts

```diff
@@ -31,26 +40,33 @@ export type ApplicationPullRequestFile = {
     client_contract_address: string
   }
   history: {
-    [key: string]: number|null
-  }
-  audit_outcomes: {
-    [key: string]: [string, string]
+    [key: string]: string
   }
+  audits: AuditCycle[]
   old_allocator_id: string
 }
 
 
-
 export async function mapApplicationToPullRequestFile(application: DatacapAllocator): Promise<ApplicationPullRequestFile>{
-  const lifeCycle = (application.applicationInstructions || []).reduce(
-    (acc, instruction, index) => {
-      const timestamp = instruction.timestamp ? instruction.timestamp.toString() : ''
-      acc[`Audit ${index + 1}`] = [timestamp, instruction.datacap_amount.toString()]
-      return acc
-    },
-    {} as { [key: string]: [string, string] },
-  )
+  
+  console.log("application.applicationInstructions", application.applicationInstructions)
```

Comment:
Left here while it beds in. We need a HUUUUGE cleanup PR soon to clean up a lot of this debt.